src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  margin: 0, auto;
+
```

Comment:
First of all corrent statement is `margin: 0 auto;` and you can remove it - it has no effects